pyseir/parameters/parameter_ensemble_generator.py

```diff
@@ -1,6 +1,13 @@
 import numpy as np
 import pandas as pd
 from pyseir import load_data
+from libs.datasets import FIPSPopulation
+from libs.datasets import DHBeds
+import us
+
+
+beds_data = DHBeds.local().beds()
+population_data = FIPSPopulation.local().population()
```

Comment:
Can you move this to the __init__ function? .I'd prefer not to have any side effects files on import

Reply:
I agree except that it is detrimental to performance since these loads are heavyweight... Maybe I can use globals which is almost as ugly, but fewer side effects.

Reply:
yeah, fine with that solution for now, but we should try to figure out in the run structure how to just define this once and pass it in where needed.

Reply:
Can we add that caching to the data source abstraction itself? Otherwise it seems the code is needlessly complex given that we want to load these and other metadata in several places in a given run.